configurators/cache/cache-interface.js

```diff
@@ -86,6 +86,17 @@ class CacheInterface {
         }
     }
 
+    setHash(name, field, value, callback) {
```

Comment:
is there no ttl with hash's?

Reply:
also, can you change `field` to `key`?

Reply:
yep, no ttl for hashes. Will rename field to key.